admin/test/test_acceptance.py

```diff
@@ -205,3 +207,19 @@ def test_non_json_message(self):
             )],
             self._convert(NON_JSON_JOURNAL_EXPORT),
         )
+
+
+class CommonOptionsParser(TestCase):
+    """
+    Tests for ``CommonOptions``
+    """
+    def test_invalid_version(self):
+
+        arg_options = "--distribution ubuntu-14.04 --provider AWS " \
+            "--config-file acceptance.yml " \
+            "flocker-version invalid-version"
+        common_options = CommonOptions(arg_options)
```

Comment:
`CommonOptions.__init__` expects an argument `top_level`, not the unparsed arguments.